### PR TITLE
Updated bee host to point to new gateway url, updated all packages to…

### PR DESCRIPTION
**Pull request by ameer-clara**
Updated bee host to point to new gateway url, updated all packages to latest, removed react intense as it depends on react 17

```The `duplex` member must be specified for a request with a streaming body```
^ issue seems to be a chrome bug afflicting the latest version 105
https://github.com/sindresorhus/ky/issues/450
https://bugs.chromium.org/p/chromium/issues/detail?id=1360943&q=duplex&can=2#makechanges

Tested successfully on Safari -> Develop -> Disable Cross-Origin Restrictions

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,34 +3,30 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ethersphere/bee-js": "^3.3.1",
+    "@ethersphere/bee-js": "^5.0.0",
```
Comment (codiak) [inline client/package.json:6]: How did we miss v4?? 😳

File: client/src/components/tweet/tweet.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default function TweetCard(props: { tweet: Tweet; account: any; profile:
       .filter((m) => m.type === "photo")
       .map((photo, index) => {
         const { media_url_https } = photo;
-        return <ReactIntense key={index} src={mediaDataUrl(media_url_https)} />;
+        return <img key={index} alt="" src={mediaDataUrl(media_url_https)} />;
```
Comment (codiak) [inline client/src/components/tweet/tweet.tsx:35]: This is a pretty big hit to usability when browsing tweets. I'm guessing the component isn't up to date?

File: client/src/process/Beejs.tsx [hunk 3]
```diff
@@ -35,18 +35,22 @@ export class Beejs {
...
+  //   "https://gateway-proxy-bee-2-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-3-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-4-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-5-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-6-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-7-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-8-0.gateway.ethswarm.org",
+  //   "https://gateway-proxy-bee-9-0.gateway.ethswarm.org",
+  //   // "https://bee-10.gateway.ethswarm.org", // this one has cors policy enabled
+  // ];
```
Comment (codiak) [inline client/src/process/Beejs.tsx:53]: Might as well clear these out now.

**Review by codiak — COMMENTED** [review]
This looks good, an exciting step forward. We should resolve the CORS issue before merging - maybe we can have a /node route or something along those lines?

File: client/src/components/tweet/tweet.tsx (hunk 2 above)
Comment (ameer-clara) [reply client/src/components/tweet/tweet.tsx:35]: Thanks for pointing this out again, it seems the latest version of ReactIntense had the issue, rolling back to the older version solves it.

File: client/src/process/Beejs.tsx (hunk 3 above)
Comment (ameer-clara) [reply client/src/process/Beejs.tsx:53]: Good call.

**Comment by ameer-clara** [comment]
> This looks good, an exciting step forward. We should resolve the CORS issue before merging - maybe we can have a /node route or something along those lines?

https://gateway.fairdatasociety.org (the bee node we are using) has to loosen its ```cors-allowed-origins: "*"```
I've reached out to the team, as previously they had servers where Cross-Origin Restrictions were disabled.

Our bee node (http://52.5.113.239:1633) is up with no cross origin restrictions, until we figure out how to make it behave like the gateway (they have a a special way of handling postage stamps), suggest we deploy this version.

**Comment by ameer-clara** [comment]
> This looks good, an exciting step forward. We should resolve the CORS issue before merging - maybe we can have a /node route or something along those lines?

https://gateway.fairdatasociety.org (the bee node we are using) has to loosen its ```cors-allowed-origins: "*"```
I've reached out to the team, as previously they had servers where Cross-Origin Restrictions were disabled.

Our bee node (http://52.5.113.239:1633) is up with no cross origin restrictions, until we figure out how to make it behave like the gateway (they have a a special way of handling postage stamps), suggest we deploy this version.

File: client/src/components/tweet/tweet.tsx (hunk 2 above)
Comment (ameer-clara) [reply client/src/components/tweet/tweet.tsx]: Opened an issue on their project
https://github.com/brycedorn/react-intense/issues/30

**Review by codiak — APPROVED** [review]

**Pull request merged**